test/jdk/java/lang/StrictMath/LogTests.java

```diff
@@ -68,6 +68,10 @@ static int testLog() {
             {0x1.0000000744b3ap632,  0x1.b611ab2bd53cep8},
             {0x1.000000037d81fp766,  0x1.0979b1dbc4a42p9},
             {0x1.000000024028p991,   0x1.577455642bb92p9},
+            // Empirical worst-case points
+            {0x1.0ffea3878db6bp+0,   0x1.f07a0cca521fp-5},
+            {0x1.490af72a25a81p-1,  -0x1.c4bf7ae48f078p-2},
+            {0x1.69e7aa6da2df5p-1,  -0x1.634508c9adfp-2},
```

Comment:
There are no libraries that have worse errors than OpenLibm, so I'm wondering what these values are good for?

Reply:
> There are no libraries that have worse errors than OpenLibm, so I'm wondering what these values are good for?

When I was working on updating the worst-case tests for Math, I would check the input values in Math.foo() and StrictMath.foo() and if they differed pick the smaller one as the reference value. The those inputs that differ, the input is a fingerprint marker for FDLIBM and I added cases to the corresponding StrictMath test.